Documentation/Signoffs/sensor_subsystem.md

```diff
@@ -0,0 +1,149 @@
+
+# Sensor Subsystem
+
+## **Function:**
+
+The goal of this subsystem is to accurately and quickly gather several data points to determine the
+distance, speed, and position of the target as it slides down the fishing line.
+
+## **Constraints:**
+
+| No. | Constraint                                                                                              | Origin            |
+| --- | ------------------------------------------------------------------------------------------------------- | ----------------- |
+| 1   | The sensor shall be supplied 5 V via USB from the Jetson Nano processor                                 | System Constraint |
+| 2   | The sensor shall be able to retrieve at least 2 data points within 1.95 s in order to calculate speed   | System Constraint |
```

Comment:
This constraint does not allow you to actually have time to aim or decide when to fire. This is not a tight enough constraint.